docs/faq/how_do_i_include_multiple_files.md

```diff
@@ -0,0 +1,3 @@
+# How do I include multiple configuration files from a directory?
+
+If you are looking to structure your configuration as a collection of include files in a directory hierarchy you can use the [kumo.glob](../reference/kumo/glob.md) function to build out the appropriate include string.
```

Comment:
I think we should either strike this entry, or replace it with example(s).  Is this about including multiple helper data files? It's not clear what sort of configuration it is referring to!